stubs/xmltodict/xmltodict.pyi

```diff
@@ -0,0 +1,34 @@
+from _typeshed import SupportsRead, SupportsWrite
+from codecs import StreamReaderWriter, StreamWriter
+from collections import OrderedDict
+from typing import IO, Any, AnyStr, Mapping, overload
+
+class ParsingInterrupted(Exception): ...
+
+def parse(
+    xml_input: AnyStr | SupportsRead[bytes],
```

Comment:
The docstring states:

> `xml_input` can be either a `string`, a file-like object.

But the source code also looks like it works fine with `bytes` inputs.

So based on all that, I would think the annotation should be something along the lines of:

```suggestion
    xml_input: str | bytes | SupportsRead[bytes],
```

(`AnyStr` is not appropriate here because there is no type dependency in this function.)